guides/common/modules/proc_configuring-the-project-mcp-server.adoc

```diff
@@ -0,0 +1,50 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-the-{project-context}-mcp-server"]
+= Configuring the {Project} MCP server
+
+Deploy the {Project} MCP server as a container and make it accessible to your MCP client.
+
+.Procedure
+ifndef::satellite[]
+. Pull the MCP container image from the registry.
+For example, to pull the latest image:
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman pull quay.io/foreman/foreman-mcp-server:latest
+----
+endif::[]
+ifdef::satellite[]
+. <TBD>
+endif::[]
+. Optional: Verify that the image is in your local Podman storage:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman images
+----
+. Run the container, specifying its connection to your {ProjectServer}.
+For example:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ podman run --interactive --tty --publish localhost:8080:8080 foreman-mcp-server \
+  --foreman-url https://_{foreman-example-com}_ \
+  --host 0.0.0.0 \
+  --port 8080 \
+  --transport stdio
+----
++
+This example command uses the following notable options:
++
+* `localhost:8080:8080` publishes port 8080 from the container to port 8080 on the `localhost` interface of the host
+* `--host 0.0.0.0` binds the MCP server inside the container to all available network interfaces, which makes the server reachable for port forwarding
+* `port 8080` ensures that the MCP server listens on this port inside the container
+
+.Next steps
+* Configure your MCP client and connect it to the MCP server.
+<TBD> Is there anything we should tell users about this step? For example, anything on what information to provide to the MCP client to enable it to authenticate to Foreman? </TBD>
+
+.Additional resources
+* `podman-run(1)` man page on your system
+* link:https://modelcontextprotocol.io/specification/2025-06-18/basic/transports[_Transports_ in the Model Context Protocol documentation]
```

Comment:
This is really unfortunate: There is no way to link to a "latest" version of the Transports page. So I can either link to this version tied to a specific date, or the modelcontextprotocol.io introduction page.